core/src/test/java/org/testcontainers/DaemonTest.java

```diff
@@ -0,0 +1,56 @@
+package org.testcontainers;
+
+import org.junit.Test;
+import org.rnorth.visibleassertions.VisibleAssertions;
+import org.testcontainers.containers.GenericContainer;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class DaemonTest {
```

Comment:
Maybe worth a comment here to explain why there's a main method, roughly what the test does?
People might jump to conclusions, perhaps!